netlify.toml

```diff
@@ -5,7 +5,7 @@
 
 # The Production Context applies to the branch you have configured as main branch to deploy:
 [context.production.environment]
-  HUGO_VERSION = "0.53"
+  HUGO_VERSION = "0.58.3"
```

Comment:
We can't update Hugo version at this moment, please see https://github.com/salesagility/SuiteDocs/issues/342